install.sh

```diff
@@ -66,9 +78,12 @@ set -euo pipefail
         break
       fi
     done
+    if [[ -z "$bin_path" || -w $bin_path ]]; then
```

Comment:
I think the `-w` condition is wrong here and shouldn't be needed

We can just do:

```suggestion
    if ! [[ -z "$bin_path"  ]]; then
```